duckduckgo/iOS
# Update BrowserServicesKit to 1.1.1

Task/Issue URL: https://app.asana.com/0/414709148257752/1200304532746654/f
Tech Design URL:
CC: @brindy

**Description**:

This PR bumps the BrowserServicesKit dependency to 1.1.1.

It also applies the latest Xcode build setting suggestions (`CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER`).

**Steps to test this PR**:
1. Run the app, run tests, etc.
1. Check that GRDB and SQLCipher do not appear in the build logs



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
